react/StoreWrapper.js

```diff
@@ -30,49 +30,23 @@ const systemToCanonical = ({ canonicalPath }) => {
   }
 }
 
-class StoreWrapper extends Component {
-  static propTypes = {
-    runtime: PropTypes.shape({
-      amp: PropTypes.boolean,
-      prefetchDefaultPages: PropTypes.func,
-      culture: PropTypes.shape({
-        country: PropTypes.string,
-        locale: PropTypes.string,
-        currency: PropTypes.string,
-      }),
-      route: PropTypes.shape({
-        metaTags: PropTypes.shape({
-          description: PropTypes.string,
-          keywords: PropTypes.arrayOf(PropTypes.string),
-        }),
-        title: PropTypes.string,
-      }),
-    }),
-    children: PropTypes.element,
-    push: PropTypes.func,
-    data: PropTypes.shape({
-      loading: PropTypes.bool,
-      manifest: PropTypes.shape({
-        // eslint-disable-next-line @typescript-eslint/camelcase
-        theme_color: PropTypes.string,
-        icons: PropTypes.arrayOf(
-          PropTypes.shape({
-            src: PropTypes.string,
-            type: PropTypes.string,
-            sizes: PropTypes.string,
-          })
-        ),
-      }),
-    }),
-  }
+const StoreWrapper = ({ children }) => {
```

Comment:
That diff

![image](https://user-images.githubusercontent.com/284515/68774592-1af72480-060c-11ea-9723-4f3689300c07.png)